libraries/stdlib/samples/test/samples/misc/tuples.kt

```diff
@@ -20,4 +20,23 @@ class Tuples {
         assertPrints(c, "[null]")
     }
 
+    @Sample
+    fun pairToList() {
+        val list = Pair(1, "a").toList()
```

Comment:
It would be informative to show what is the static type of `list` by using explicit type specification
```
val list: List<...> = ...
```
It's worth to add another example, where both components have the same type.